src/core/public/core_app/status/lib/load_status.ts

```diff
@@ -144,8 +179,15 @@ export async function loadStatus({
   return {
     name: response.name,
     version: response.version,
-    statuses: response.status.statuses.map(formatStatus),
-    serverState: formatStatus(response.status.overall).state,
+    statuses: [
+      ...Object.entries(response.status.core).map(([serviceName, status]) =>
+        formatStatus(`core:${serviceName}@${response.version.number}`, status)
```

Comment:
IMO we should remove the version number from the core services as well. I think it's confusing and unnecessary.